src/dev/src/app/control-warning/control-warning.demo.html

```diff
@@ -0,0 +1,171 @@
+<h1>Control Warning</h1>
```

Comment:
this should go into the "website" project, where the documentation resides.
the "dev" project is just a playground for developers.